src/card/info-status/info-status-card.html

```diff
@@ -0,0 +1,19 @@
+<div class="card-pf card-pf-info-status"
+     ng-class="{'card-pf-accented': $ctrl.shouldShowTopBorder}">
+  <div class="info-image-container">
+    <img ng-if="$ctrl.status.iconImage" ng-src="{{$ctrl.status.iconImage}}" alt=""
+         class="info-img"/>
+    <span class="info-icon {{$ctrl.status.iconClass}}"></span>
+  </div>
+  <div>
+      <h2 class="card-pf-title" ng-if="$ctrl.status.title">
+        <a href="{{$ctrl.status.href}}" ng-if="$ctrl.status.href">
+          <span class="">{{$ctrl.status.title}}</span>
+        </a>
+        <span ng-if="!$ctrl.status.href">
+            <span class="">{{$ctrl.status.title}}</span>
+          </span>
+      </h2>
+    <p ng-bind-html="$ctrl.trustAsHtml(item)" ng-repeat="item in $ctrl.status.info track by $index"></p>
```

Comment:
I think you could just have:
```
<p ng-bind-html="item" ...
```

Reply:
Nah, we need it, otherwise `Error: [$sce:unsafe] Attempting to use an unsafe value in a safe context.`   If you notice, eerily similar to https://github.com/AllenBW/angular-patternfly/blob/71d481d5def61ec759b19c8a1d0188c432dd3697/src/notification/toast-notification.html#L36 😏

Reply:
I think we should make this opt in as well. Treat as text unless html is specified.